linode_api4/linode_client.py

```diff
@@ -871,6 +871,11 @@ def ips(self, *filters):
     def ipv6_ranges(self, *filters):
         return self.client._get_and_filter(IPv6Range, *filters)
 
+    def ip_ranges_list(self):
+        result = self.client.post("/networking/ipv6/ranges", model=self)
+
+        return [IPv6Range(self.client, r["range"]) for r in result["data"]]
+
```

Comment:
Is there any reason that makes `_get_and_filter` not suitable here? And what does it do? It looks similar to `ipv6_ranges`...

Reply:
Yes you're right! The endpoint was listed as one of the ones I needed to add in the ticket description and I didn't realize there was an existing function for it. I will remove the duplicated fucntion.